### ISSUE TITLE
Testing Search Inside Results Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4637]
# Page to Test:

Search Inside Results - https://openlibrary.org/search/inside?q=The+Crucible&mode=ebooks&has_fulltext=true

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly All the book covers present on this page have alt attr so we can close this issue.
